packages/genesys-spark-components/src/components/stable/gux-breadcrumbs/breadcrumb-item/gux-breadcrumb-item.scss

```diff
@@ -144,11 +145,9 @@ $temp-focus-ring-corner-radius: 4px;
       }
 
       &:focus-visible {
-        border-radius: calc(
-          $temp-focus-ring-corner-radius - $temp-focus-ring-offset
-        );
-        outline: $temp-focus-ring-width solid var(--gse-semantic-color-focus);
-        outline-offset: $temp-focus-ring-offset;
+        @include focus.gux-focus-ring;
+
+        border-radius: var(--gse-semantic-focusRing-cornerRadius);
```

Comment:
@katie-bobbe-genesys Do you know why this is not included in the mixin?

Reply:
I think it's because we want to match the border-radius of the focused element itself. It would look weird if there was a different border radius value when the element is focused.

Reply:
That is what I saw as well. Lots of components with different border radii and some of them have specified focus ring border radii.

Reply:
We should report this issue to UX.

Reply:
Is this an issue? Things like the gux-toggle or gux-avatar would definitely not have the same border radius as an input for example, but they still can be tabbed to.